src/import/engine.rs

```diff
@@ -279,6 +289,22 @@ fn tune_dboptions_for_bulk_load(opts: &DbConfig) -> (DBOptions, CFOptions) {
     (db_opts, CFOptions::new(CF_DEFAULT, cf_opts))
 }
 
+fn tune_dboptions_for_import(_: &DbConfig) -> (DBOptions, CFOptions) {
+    let mut db_opts = DBOptions::new();
+    db_opts.enable_statistics(false);
+    // We don't need any background threads to do compaction or flush job.
+    db_opts.set_max_background_jobs(0);
+
+    let mut cf_opts = ColumnFamilyOptions::new();
+    // We don't use memtable for import at all, so we can reduce write buffer
+    // size to save memory.
+    cf_opts.set_write_buffer_size(ReadableSize::mb(1).0 as u64);
```

Comment:
I think even if you set a large size here, RocksDB will not allocate the whole write buffer at once, so I think it is not necessary to tune here.